### ISSUE TITLE
Undefined values are not presented

### ISSUE THREAD

**Issue by mdziekon** [chibicode/react-json-tree#14]
It appears that ``undefined`` values are not shown at all in the tree, even when our objects have explicitly created keys with this value. But they are being added to the "keys" counter label, so this looks kind of weird - seeing, say, "3 keys" in the counter, but expanded view shows only 2 of them which have defined values.

Looking at the codebase, it's probably a simple fix, so I'll provide a PR with a patch.
**Edit**: PR: #15 

#### Example that shows the problem:

With the following object:
```javascript
{
    object: {
        foo: 'bar',
        baz: undefined
    },
    test: {
        my: undefined
    }
}
```

**The actual outcome** is shown at the following screenshot:
![json-pre-fix](https://cloud.githubusercontent.com/assets/4425221/10528149/baa5482c-7394-11e5-898e-24708dcbfd59.png)

**The expected outcome** should be as on the next screenshot:
![json-fix](https://cloud.githubusercontent.com/assets/4425221/10528295/affbe92a-7395-11e5-9702-99a381c9c6fb.png)

**Comment by redexp** [comment]
Any progress on this issue? Same problem with `function` type which can look like `component: function User()`

**Comment by chibicode** [comment]
@mdziekon @redexp should be fixed in 0.3.0!
